src/tools/meta-dump/MetaDumpTool.cpp

```diff
@@ -52,8 +54,9 @@ class MetaDumper {
       }
     }
     {
-      LOG(INFO) << "Partition info";
-      prefix = "__parts__";
+      LOG(INFO) << "------------------------------------------\n\n";
+      LOG(INFO) << "Partition info::";
```

Comment:
why double colon?

Reply:
It will let the split boundary more obvious.